addons/mail/models/ir_attachment.py

```diff
@@ -18,23 +21,22 @@ def _post_add_create(self):
     def register_as_main_attachment(self, force=True):
         """ Registers this attachment as the main one of the model it is
         attached to.
+
+        :param bool force: if set, the method always updates the existing main attachment
```

Comment:
Could be great to expand a bit the commit message, telling it is code cleaning done during move of main attachment into its own mixin, that no functional changes should occur with this commit, that we mainly do pre-check for early returns, that some variable renaming is done to ease understanding, ...